ush/parsing_namelists_MOM6.sh

```diff
@@ -41,6 +41,10 @@ cat input.nml
 # ---------
 # Prepare local variables for use in MOM_input.IN from UFSWM
 # The ones already defined are left commented as a reminder
+# == MOM options to start from coarsed grained restarts, set to off by default
+local MOM6_INIT_FROM_Z=${MOM6_INIT_FROM_Z:-True}
```

Comment:
This should be False to match the current default
```suggestion
local MOM6_INIT_FROM_Z=${MOM6_INIT_FROM_Z:-False}
```

Reply:
The default is True, see line 1016

https://github.com/NeilBarton-NOAA/ufs-weather-model/blob/develop/tests/default_vars.sh

It is set to False for use with the coarse grained/interpolated MOM6 restarts.

Reply:
Okay, they have the wrong comment in the template then: https://github.com/ufs-community/ufs-weather-model/blob/develop/tests/parm/MOM_input_050.IN#L309